cmake/Modules/BuildCommon.cmake

```diff
@@ -64,6 +64,7 @@ set(KATANA_USE_LCI OFF CACHE BOOL "Use LCI network runtime instead of MPI")
 set(KATANA_NUM_TEST_THREADS "" CACHE STRING "Maximum number of threads to use when running tests (default: min(number of physical cores, 8))")
 set(KATANA_AUTO_CONAN OFF CACHE BOOL "Automatically call conan from cmake rather than manually (experimental)")
 set(KATANA_NUM_DOC_THREADS "" CACHE STRING "Maximum number of threads to use when reading / writing with Sphinx (default: min(number of physical cores, 4))")
+set(KATANA_USE_DEPRECATED_ERROR OFF CACHE BOOL "If set, when treating warnings as errors also treat deprecated declarations as errors ")
```

Comment:
darn trailing space

Reply:
hahahaha. I hate it when that happens. I didn't see it thought. You can pretend we didn't notice.